deploy/kubeturbo/values.yaml

```diff
@@ -73,6 +73,16 @@ HANodeConfig:
 #   namespace: ""
 #   workloadController: ""
 
+# The featureGates property defines a map of string-to-boolean values that provides users with a mechanism 
+# to enable # features that are disabled by default. For a list of supported feature gates, see 
+# https://github.com/turbonomic/kubeturbo/blob/master/pkg/features/features.go
+# featureGates:
+#   PersistentVolumes: true
+#   ThrottlingMetrics: false
+#   HonorAzLabelPvAffinity: true
+#   GitopsApps: false
+#   GoMemLimit: true
```

Comment:
* Shall we specify the current default values to these features?
* For the following:
  > provides users with a mechanism to enable features that are disabled by default
  
  it is also true that we can disable features that are enabled by default ... So may be just say provides users with a mechanism to enable or disable a feature?

Reply:
Sure, I can add the default values.

Reply:
Updated description and added default values.